### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by lukaszdebowski**
### Changes:
- Updated dependencies of the project (needed for Lifeline 😄 )
- Added `positiveNumber` validator
- Added `largerThan` validator

### CODE REVIEW

File: lib/src/validator.dart [hunk 1]
```diff
@@ -144,8 +144,8 @@ abstract class Validators {
         }
       };
 
-  /// Returns an [errorMessage] when the value of a field is not a valid inteager.
-  static Validator inteager({
+  /// Returns an [errorMessage] when the value of a field is not a valid integer.
+  static Validator integer({
```
Comment (kamilklyta) [inline lib/src/validator.dart:148]: <p>Nice :iq-200:!</p>

File: lib/src/validator.dart [hunk 2]
```diff
@@ -273,4 +273,20 @@ abstract class Validators {
...
+  ///Returns [errorMessage] when field value is not a [num] > 0.
+  static Validator positiveNumber({
+    @required String errorMessage,
+  }) =>
+      (String value) {
+        if (!value.isDouble) return errorMessage;
+
+        final number = num.parse(value);
+
+        if (number <= 0) {
```
Comment (kamilklyta) [inline lib/src/validator.dart:286]:
```suggestion
        return number.isNegative :  errorMessage : null;
```

**Review by kamilklyta — CHANGES_REQUESTED** [review]

File: lib/src/validator.dart (hunk 2 above)
Comment (lukaszdebowski) [reply lib/src/validator.dart]: I've modified your code to also return error when the value is 0 (we only want positive as the name suggests)

**Review by kamilklyta — APPROVED** [review]

File: lib/src/validator.dart [hunk 3]
```diff
@@ -273,4 +273,25 @@ abstract class Validators {
     TextEditingController controller,
   }) =>
       (String text) => controller.value.text != text ? errorMessage : null;
+
+  ///Returns [errorMessage] when field value is not a [num] > 0.
+  static Validator positiveNumber({
+    @required String errorMessage,
+  }) =>
+      (String value) {
+        if (!value.isDouble) return errorMessage;
```
Comment (kamilklyta) [inline lib/src/validator.dart:282]:
```suggestion
        if (!value.isNumeric) return errorMessage;
```

File: lib/src/validator.dart [hunk 4]
```diff
@@ -273,4 +273,25 @@ abstract class Validators {
...
+        return number.isNegative ? errorMessage : null;
+      };
+
+  ///Returns [errorMessage] when field value is not larger then [threshold].
+  static Validator largerThan({
+    @required String errorMessage,
+    double threshold = 0.0,
+  }) =>
+      (String value) {
+        if (!value.isDouble) return errorMessage;
```
Comment (kamilklyta) [inline lib/src/validator.dart:293]:
```suggestion
        if (!value.isNumeric) return errorMessage;
```

File: lib/src/validator.dart [hunk 5]
```diff
@@ -273,4 +273,25 @@ abstract class Validators {
...
+        final number = num.parse(value);
+        return number.isNegative ? errorMessage : null;
+      };
+
+  ///Returns [errorMessage] when field value is not larger then [threshold].
+  static Validator largerThan({
+    @required String errorMessage,
+    double threshold = 0.0,
```
Comment (kamilklyta) [inline lib/src/validator.dart:290]: I think that it looks weird when `largerThan` method does not have any parameter provided 🤔.
I think that there shouldn't be any default value.
```dart
Validators.largerThan(errorMessage: "Number must be larger than... than what?");
```

**Review by kamilklyta — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ dependencies:
   validator:
     git: 
       url: https://github.com/HTD-Health/validators.git
-      ref: 0.1.0
+      ref: 0.2.0
```
Comment (kamilklyta) [inline README.md:12]: :iq-200:

**Review by kamilklyta — APPROVED** [review]

**Pull request merged**